### ISSUE TITLE
Minimize to tray

### ISSUE THREAD

**Issue by eexil** [lakoliu/Furtherance#100]
Would it be possible to enable the app to minimize to tray?
ATM when you click minimize it will go out of view but it will take up space in the window list in the panel.
It would be better if it just sat in the sys-tray and one could bring it back up in view for there.
![image](https://user-images.githubusercontent.com/16208522/210247162-60d569a5-9b82-4f5b-807d-d7cb281fd0b6.png)

Thank you!

**Comment by lakoliu** [comment]
I think that would be a great feature if it were an option in the settings. I don't want to make it the default since GNOME doesn't have a system tray for app icons anymore without using an extension. I'll look into doing this. Thanks for the suggestion!

**Comment by erazemk** [comment]
Seems this is now possible without an extention in GNOME 44 ([gnome-shell #2624](https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2624)).

**Comment by corebots** [comment]
Would be great if it could show in the top bar, similar to when doing screen recording! :) 
[Screencast from 2023-06-20 11-15-12.webm](https://github.com/lakoliu/Furtherance/assets/12252430/c5834522-14b0-4347-a5d4-b1072d2a57a5)

**Comment by isle9** [comment]
Any plans for this? The app is unfortunately unusable for me without the close to tray option.

**Comment by lakoliu** [comment]
Yes, I do plan on implementing this, but I have another project I'm actively working on at the moment, so I can't say how long it will be before I get to it. If anyone else wants to work on it in the meantime that would be great!

**Comment by lakoliu** [comment]
I've looked more into this and as far as I can tell there is still no direct way of doing this in GNOME without creating a shell extension. I don't write Javascript so that is beyond me at the moment, but if someone else would like to build it that would be awesome! As far as the Background Apps panel goes, @erazemk, that looks cool, but I'm also not sure how to integrate Furtherance with that using gtk4-rs. It also looks like a Javascript shell extension (maybe?). Also, that is nice to close it and launch it from the background, but it wouldn't show any cool toolbar timer like the video @corebots shared.

**Comment by tomaskikutis** [comment]
Having it in tray is what I miss the most. I would like an option not to show tracked time. For me it'd be distracting. I'd only want an indication whether something is being tracked or not.

**Comment by RCPhysics** [comment]
Would love to have a version that minimizes to the menu bar in MacOS.
